README.md

```diff
@@ -38,6 +38,19 @@ for `pip`.
 
 Ensure that you have [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) installed.
 
+##### From conda-forge
+
+You can install [pathsimanalysis from the conda-forge channel](https://anaconda.org/conda-forge/pathsimanalysis) with
```

Comment:
Oops sorry for the double post - looks like the previous one got added to a commit instead of the PR.

My question was whether this might conflict with the "from source" comment above.

Reply:
Yes, I guess, but the pip installation (which may use wheels) was already misleading. I just changed it into package/source.